### PR TITLE
fix: disable polling in previews

### PR DESCRIPTION
**Pull request by TylerBarnes**
Don't send previews for published posts, instead use the regular save post webhook logic to post to the preview instance.

### CODE REVIEW

**Comment by TylerBarnes** [comment]
This is ready for review, but not to be merged. Just doing some manual testing on it right now

**Comment by TylerBarnes** [comment]
The Gatsby side of this is here https://github.com/gatsbyjs/gatsby-source-wordpress-experimental/pull/317

**Comment by TylerBarnes** [comment]
This is ready to be reviewed and merged!

File: src/Admin/Preview.php [hunk 1]
```diff
@@ -501,6 +501,14 @@ public function post_to_preview_instance( $post_ID, $post ) {
 
 		$original_post = get_post( $post->post_parent );
 
+		$this_is_a_publish_not_a_preview =
```
Comment (TylerBarnes) [inline src/Admin/Preview.php:504]: @jasonbahl can you think of any problems with this? From my testing it works great. If we don't do this, we'll send a preview + a save post action at the same time (both to the preview instance).

**Review by jasonbahl — APPROVED** [review]

**Pull request merged**